README.md

```diff
@@ -1,73 +1,42 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
-
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+This is my first Microverse Hello world project whose main focus is to test my understanding in github,githubflows,version controll and to demonstrate the ability to submit a project for code review.
 
+Built With:
+*HTML
+*CSS
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
+*Obove the list list of files click Code
+*To clone the repository using HTTPS, under "Clone with HTTPS", click the copy button . To clone the repository using an SSH key, including a certificate issued by your organization's SSH certificate authority, click Use SSH, then click the copy button . To clone a repository using GitHub CLI, click Use GitHub CLI, then click the copy button .
+*Open Git Bash.
+*Change the current working directory to the location where you want the cloned directory.
+*Type git clone, and then paste the URL you copied earlier.
+*Press Enter to create your local clone.
 
-### Install
 
-### Usage
 
-### Run tests
 
-### Deployment
+Authors
+👤 Author1
+* GitHub:https://github.com/frank1738
+* Twitter: https://twitter.com/frankhiggins08
+* LinkedIn: www.linkedin.com/in/ 
+frankline-osoro-b526ba18b
```

Comment:
- [ ] [OPTIONAL] You have done a great job implementing the social links, however, the link for LinkedIn is broken, kindly fix it or remove it. This will enable visitors to contact you via LinkedIn and take a look at your profile.